webviz_subsurface/_datainput/opm_init_io.py

```diff
@@ -338,7 +393,7 @@ def create_dry_gas(self, raw: EclPropertyTableRawData) -> List[PvxOBase]:
             values = []
 
             # PKey  Inner   C0  C1          C2          C3
-            # Rs    Po      1/B 1/(B*mu)    d(1/B)/dPo  d(1/(B*mu))/dPo
+            # Pg    Rv      1/B 1/(B*mu)    d(1/B)/dPg  d(1/(B*mu))/dPg
```

Comment:
Guess this is the same question as for dead oil: do you have a Rv here? But I get a feeling now that it exists, but is constant 0 🙃